imports/plugins/core/orders/client/templates/workflow/shippingSummary.js

```diff
@@ -34,10 +34,14 @@ Template.coreOrderShippingSummary.onRendered(function () {
 
   if (order.workflow) {
     if (order.workflow.status === "coreOrderCreated") {
-      order.workflow.status = "coreOrderCreated";
       Meteor.call("workflow/pushOrderWorkflow", "coreOrderWorkflow", "coreOrderCreated", order);
     }
   }
+  if (order.workflow) {
```

Comment:
Since there is a `if (order.workflow) {` above you can put the content of this block inside it. I'm also thinking you can eliminate one of the `if blocks` like we just did.